mathics/builtin/arithmetic.py

```diff
@@ -86,6 +86,13 @@
 
 @lru_cache(maxsize=4096)
 def call_mpmath(mpmath_function, mpmath_args, prec=None):
+    """
+    calls the mpmath_function with mpmath_args parms
+    if prec=None, use floatting point arithmetic.
+    Otherwise, work with prec bits of precision.
+    """
+    # TODO: rocky, please help me with the annotations
```

Comment:
* `mpmath_function`  is a `Callable` 
* `mpmath_args` is a `tuple` 
* `prec` is `Optional[int]`